### ISSUE TITLE
Firehose crash loop due to "Commit.Commit got an unexpected keyword argument"

### ISSUE THREAD

**Issue by emilyhunt** [emilyhunt/bluesky-astronomy-feeds#15]
The firehose got stuck in a crash loop this morning due to the following bug.

It's also really bad that it can crash loop! This shouldn't happen.

```
[firehose] [2023-08-30 07:59:03] INFO:server.firehose:Trying to re-open database connection (as this is a common issue...)
[firehose] [2023-08-30 07:59:03] INFO:server.firehose:Exception encountered in on_message_handler! Commit.Commit got an unexpected keyword argument 'rev'
[firehose] [2023-08-30 07:59:03] Traceback (most recent call last):
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/xrpc_client/models/utils.py", line 106, in _get_or_create
[firehose] [2023-08-30 07:59:03]     model(**model_data)
[firehose] [2023-08-30 07:59:03] TypeError: Commit.__init__() got an unexpected keyword argument 'rev'
[firehose] [2023-08-30 07:59:03] 
[firehose] [2023-08-30 07:59:03] The above exception was the direct cause of the following exception:
[firehose] [2023-08-30 07:59:03] 
[firehose] [2023-08-30 07:59:03] Traceback (most recent call last):
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/firehose/client.py", line 141, in _process_message_frame
[firehose] [2023-08-30 07:59:03]     self._on_message_callback(frame)
[firehose] [2023-08-30 07:59:03]   File "/workspace/server/firehose.py", line 89, in on_message_handler
[firehose] [2023-08-30 07:59:03]     commit = parse_subscribe_repos_message(message)
[firehose] [2023-08-30 07:59:03]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/firehose/__init__.py", line 52, in parse_subscribe_repos_message
[firehose] [2023-08-30 07:59:03]     return get_or_create_model(message.body, model_class)
[firehose] [2023-08-30 07:59:03]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/xrpc_client/models/utils.py", line 124, in get_or_create_model
[firehose] [2023-08-30 07:59:03]     model_instance = get_or_create(model_data, model)
[firehose] [2023-08-30 07:59:03]                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/xrpc_client/models/utils.py", line 81, in get_or_create
[firehose] [2023-08-30 07:59:03]     raise e
[firehose] [2023-08-30 07:59:03]   File "/workspace/.heroku/python/lib/python3.11/site-packages/atproto/xrpc_client/models/utils.py", line 78, in get_or_create
[firehose] [2023-08-30 07:59:03]     return _get_or_create(model_data, model, strict=strict)
```

**Comment by emilyhunt** [comment]
Crash loop continues even when the feed is restarted (oh noooooooo.) Needs a fix!

**Comment by emilyhunt** [comment]
Fixed by b18cbe4a4d7f974ab3f65700ec10a29e9b2373b7 - the issue was upstream due to a breaking change in the AT Protocol API.
